dist/src/main/java/io/camunda/application/commons/metrics/jfr/NativeMemoryMetrics.java

```diff
@@ -102,12 +112,12 @@ private void register(
       Gauge.builder(doc.getName(), reserved, AtomicLong::get)
           .description(doc.getDescription())
           .baseUnit("bytes")
-          .tags(List.of(tags))
+          .tags(Tags.of(tags).and(Tag.of(VALUE.asString(), RESERVED.value())))
```

Comment:
:thought_balloon: Can be differentiated by adding a `value=reserved` or `value=committed`. I wasn't sure about the tag name, feel free to change it.

Reply:
It's quite hard to come up with something better, especially because `reserved` does not actually mean it's allocated, so "allocationType" is wrong for example.
What about `state`?

Reply:
None of them seem good, I really don't know :smile:

Reply:
I would go with state :shrug: